### PR TITLE
Update release notes to include qualification and profiling tools

### PR DESCRIPTION
**Pull request by sameerz**
Mention qualification and profiling tools in release notes.  Once this is merged I will add this commit to branch-21.08.

### CODE REVIEW

File: docs/download.md [hunk 1]
```diff
@@ -73,6 +73,10 @@ New functionality for this release includes:
   * Range windows supporting non-timestamp order by expressions
 * Enabling large joins that can spill out of memory
 * Support for the `concat_ws` operator
+* Spark Qualification and Profiling Tools
+  * A qualification tool to look at a set of applications to determine if the RAPIDS Accelerator for
+    Apache Spark is a good fit
+  * A profilng tool to generate information used for debugging and profiling applications
```
Comment (jlowe) [inline docs/download.md:79]:
```suggestion
  * A profiling tool to generate information used for debugging and profiling applications
```

**Review by GaryShen2008 — APPROVED** [review]

File: docs/download.md [hunk 2]
```diff
@@ -73,6 +73,10 @@ New functionality for this release includes:
   * Range windows supporting non-timestamp order by expressions
 * Enabling large joins that can spill out of memory
 * Support for the `concat_ws` operator
+* Spark Qualification and Profiling Tools
```
[the remark this answers is not in the document]
Comment (jlowe) [reply docs/download.md:76]: A bit disappointed that this comment was completely ignored before the PR was merged.

[the remark this answers is not in the document]
Comment (sameerz) [reply docs/download.md:76]: > A bit disappointed that this comment was completely ignored before the PR was merged.

Sorry not sure how I missed this comment before merging, I might have thought I was responding to the comment in the related 21.08 PR - https://github.com/NVIDIA/spark-rapids/pull/2849#issuecomment-871688194 .  Asking @viadea to create documentation for the profiling tool that will reside in the gh-pages docs in 21.08.

[the remark this answers is not in the document]
Comment (sameerz) [reply docs/download.md:76]: I am happy to add a link to in gh-pages to the profiling tools readme at this point if you still think it is worthwhile.

[the remark this answers is not in the document]
Comment (jlowe) [reply docs/download.md:76]: I don't want to have the gh-pages version of the release notes and the checked-in version differ unless it's critical to fix something and we can't keep them in sync.  If we want to do something there, we should fix it in branch-21.08 and possibly pull that change into gh-pages early.  If there's no good doc to point to until 21.08 then I don't see a need to rush and fix this for 21.06.

**Pull request merged**